PeerioTechnologies/peerio-desktop
# chore: PeerUI wrapper, default a11y styles on buttons/inputs

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/12057/peerui-active-focus-feedback

#### Testing instructions
Basic keyboard navigation feedback. If you tab through elements in the app, buttons should highlight with a blue outline. If you move the mouse, the outline disappears.

This PR just changes the default styling, but doesn't change the app's own focusable elements. For example I'm not sure if the ChatList can even be tabbed through right now. That's for a more in-depth a11y sweep once designs are done.

For now the best place to test this is probably the signup process.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
